src/graph.c

```diff
@@ -1023,6 +1023,155 @@ int Graph_Zoom(const LCUI_Graph *graph, LCUI_Graph *buff, LCUI_BOOL keep_scale,
 	return 0;
 }
 
+int Graph_Zoom_Bilinear(const LCUI_Graph *graph, LCUI_Graph *buff,
```

Comment:
You should add links to some reference documents to make it easier for others to understand.